cross/arm-none-eabi-gdb/Portfile

```diff
@@ -14,3 +14,22 @@ configure.args-append \
 
 # We don't actually violate the tree.
 destroot.violate_mtree  no
+
+# Copied from devel/boost187
+set pythons_versions {2.7 3.9 3.10 3.11 3.12 3.13}
+
+set pythons_ports {}
+foreach v ${pythons_versions} {
+    lappend pythons_ports python[string map {. {}} ${v}]
+}
+foreach v ${pythons_versions} {
+    set s [string map {. {}} ${v}]
+    set p python${s}
+    set i [lsearch -exact ${pythons_ports} ${p}]
+    set c [lreplace ${pythons_ports} ${i} ${i}]
+    variant ${p} description "Build Boost.Python for Python ${v}" conflicts {*}${c} "
```

Comment:
pleas update variant description, this likely has nothing to do with Boost...

Reply:
Fixed.